package.json

```diff
@@ -24,11 +24,11 @@
     "homepage": "https://github.com/yahoo/jsx-test",
     "devDependencies": {
         "babel": "^5.0",
-        "istanbul": "^0.3.15",
         "jenkins-mocha": "^2.2.0",
         "react": "^0.14.0"
     },
     "dependencies": {
+        "istanbul": "^0.3.15",
```

Comment:
I am now using `jenkins-mocha` to run my jsx-test.  Here's my current npm script target:

*test:jenkins-mocha with coverage*
> JENKINS_MOCHA_COVERAGE=true jenkins-mocha --timeout 5000 --compilers js:babel/register tests/**/*.test.js*

*devtest:jenkins-mocha without coverage*
> jenkins-mocha --no-coverage --timeout 5000 --compilers js:babel/register tests/**/*.test.js*

I believe we can use `istanbul + jenkins-mocha + babel` command to generate the coverage for jsx-test.  If yes, we document it before dropping the dependency.

remark:  I would like to have a code coverage on the `source` directory (1.) instead of transpired `dist` directory. (2.)

1.
<img width="965" alt="screen shot 2015-11-25 at 3 07 11 pm" src="https://cloud.githubusercontent.com/assets/51471/11411602/5d9d1a44-9386-11e5-9892-3ee578637dce.png">

2.
![screen shot 2015-11-25 at 3 07 16 pm](https://cloud.githubusercontent.com/assets/51471/11411604/6220b0a8-9386-11e5-9ff3-0c1e5373ac7a.png)